pub/css/main.css

```diff
@@ -0,0 +1,371 @@
+/*! HTML5 Boilerplate v4.3.0 | MIT License | http://h5bp.com/ */
+
+html,
+button,
+input,
+select,
+textarea {
+    color: #222;
+}
+
+html {
+    font-size: 1em;
+    line-height: 1.4;
+}
+
+::-moz-selection {
+    background: #b3d4fc;
+    text-shadow: none;
+}
+
+::selection {
+    background: #b3d4fc;
+    text-shadow: none;
+}
+
+hr {
+    display: block;
+    height: 1px;
+    border: 0;
+    border-top: 1px solid #ccc;
+    margin: 1em 0;
+    padding: 0;
+}
+
+audio,
+canvas,
+img,
+video {
+    vertical-align: middle;
+}
+
+fieldset {
+    border: 0;
+    margin: 0;
+    padding: 0;
+}
+
+textarea {
+    resize: vertical;
+}
+
+.browsehappy {
+    margin: 0.2em 0;
+    background: #ccc;
+    color: #000;
+    padding: 0.2em 0;
+}
+
+
+/* ==========================================================================
+   Author's custom styles
+   ========================================================================== */
+
+/*Container Style*/
+.container {
+width: 80%; margin: auto;
+}
+
+/*Header Style*/
+header h1 {
+    text-align: center;
+    font-family: 'Droid Sans', sans-serif;
+    color: #46A4DA;
+    font-size: 2em;
+}
+
+header i {
+    color: #46A4DA;
+    border: 1ex solid #46A4DA;
+    border-radius: 50%;
+    padding: 1ex;
+    margin-bottom: 5ex;
+}
+
+header div {
+    width: 30%;
+    text-align: center;
+    margin-right: auto;
+    margin-left: auto;
+}
+
+.timeline {
+    margin: 0 auto; padding: 0;
+    list-style: none;
+    padding-top: 3em;
+}
+
+.timeline-post article {
+    display: block;
+    position: relative;
+    padding: 1ex 2em;
+}
+
+.timeline-post {
+    margin-bottom: 6em;
+}
+
+time small {
+    color: darkgray;
+}
+
+time big {
+    font-size: 1.5em
+}
+
+.timeline-post:nth-child(odd) {
+    background-color: #6CBFEE;
+    border-radius: 1em;
+}
+
+.timeline-post:nth-child(even) {
+    background-color: #46A4DA;
+    border-radius: 1em;
+}
```

Comment:
Something I realize that I didn't really cover in class:

> 'Tis better to set a default than to rely on fancy selectors.

The `:nth-child()` pseudo-class is pretty well-supported at this point, but still not _fully_ supported. A browser that _doesn't_ support it (or supports it incorrectly) will ignore these rules, effectively leaving you with white text on a white background. Better to set a default color -- light blue or dark blue -- and then override that with `:nth-child()`...